docs/aerodromes/classc/Nowra.md

```diff
@@ -4,7 +4,146 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | ID      | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | --------------| -------------- | ---------------- | --------------------------------------|
+| Nowra TMA    | | Nowra Approach | 123.500          |NW_APP        |
+| Nowra ADC    | | Nowra Tower  | 118.850         | NW_TWR        |
+| Nowra SMC   ||  Nowra Ground  | 135.850         | NW_GND        |
+| Nowra ACD    | | Nowra Delivery  | 128.350         | NW_DEL       |
+| Nowra ATIS    |   | | 125.65         | YSNW_ATIS       |
+
+## Airspace
+
+### TMA Ownership
+
+The Terminal Control Area (TMA) owns the airspace within R421 up to FL125; optionally R420F extends from FL125 to FL300.
+This airspace is for facilitating military aircraft and low-level helicopter operations.
+
+### Activation of Restricted Areas
+
+When a controller logs in as Nowra ADC (NW ADC) or TMA:
+
+1. **Announce Activation**: Announce that R421 and other applicable PRDs are active.
+2. **Activate Surrounding Areas**: Notify surround controllers to activate the restricted areas to get the outlines visible.
+3. **Issue Clearances**: Treat restricted areas like R421 as Class C/D airspace and issue clearances accordingly.
+4. **Service Levels**:
+    - Official aircraft: Class D service. (eg Military)
+    - Unofficial aircraft: Class C service. (eg Non Military)
```

Comment:
I assume this is from an official source? Would prefer that we simplify by providing a class C service to all aircraft, in line with other military SOP pages.

Reply:
Yeah the Nowra sop listed here 

https://ais-af.airforce.gov.au/sites/default/files/current-ad2/2406%20Nowra%20FIHA%20AD2%20SUPP_0.pdf

Also the ersa 

![image](https://github.com/user-attachments/assets/2be29795-a8d7-45b0-9533-3797570e40d4)